### PR TITLE
Restore UAP test tools via a package from internal feed.

### PR DESCRIPTION
**Pull request by AlexGhiondea**
/cc @weshaggard @joperezr @MattGal

### CODE REVIEW

File: external/test-runtime/XUnit.Runtime.depproj [hunk 1]
```diff
@@ -43,8 +43,16 @@
           AfterTargets="RestorePackages"
           Condition="'$(TargetGroup)'=='uap'" >
 
+    <PropertyGroup>
+      <UAPToolsPackageVersion>1.0.1</UAPToolsPackageVersion>
+      <UAPToolsPackageName>Microsoft.DotNet.UAP.TestTools</UAPToolsPackageName>
+
+      <UAPToolsFolder>$(PackagesDir)$(UAPToolsPackageName)\$(UAPToolsPackageVersion)\Tools</UAPToolsFolder>
```
Comment (weshaggard) [inline external/test-runtime/XUnit.Runtime.depproj]: I would still allow for someone to set this locally so I would add a condition for if this is empty set it.

Comment (AlexGhiondea) [inline external/test-runtime/XUnit.Runtime.depproj]: Good point. Will change!

Comment (AlexGhiondea) [inline external/test-runtime/XUnit.Runtime.depproj]: @weshaggard done.

**Comment by AlexGhiondea** [comment]
Test this

**Comment by AlexGhiondea** [comment]
@dotnet-bot test this please

**Pull request merged**